src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/validators.py

```diff
@@ -41,6 +41,24 @@ def password_validator(ns):
             raise CLIError('Please specify password in non-interactive mode.')
 
 
+def retention_validator(ns):
+    if ns.backup_retention_days:
+        val = ns.backup_retention_days
+        if val >= 7 and val <= 35:
+            pass
+        else:
+            raise CLIError('incorrect usage: --backup_retention. Range is 7 to 35 days.')
+
+
+def retention_validator2(ns):
+    if ns.backup_retention:
+        val = ns.backup_retention
+        if val >= 7 and val <= 35:
+            pass
+        else:
+            raise CLIError('incorrect usage: --backup_retention. Range is 7 to 35 days.')
```

Comment:
You can avoid having this second validator by checking for both `backup_retention` and `backup_retention_days`. 
As a better change, it would be best to change the instances of `backup_retention` to `backup_retention_days` in the function signature being used the the custom.py file.

Reply:
Updated the function and get rid of the extra validator